flocker/provision/_install.py

```diff
@@ -902,8 +871,6 @@ def provision(distribution, package_source, variants):
     """
     commands = []
 
-    if Variants.DISTRO_TESTING in variants:
```

Comment:
`DISTRO_TESTING` is still defined as a variant but it looks like it's basically no longer supported.  This might be confusing.

I suggest leaving this, leaving `task_enable_updates_testing`, but make the task always raise the unsupported distribution exception.  Hopefully we'll be able to do some testing-distribution testing for other distributions eventually.